django/contrib/admin/static/admin/css/base.css

```diff
@@ -102,8 +102,8 @@ body {
 /* LINKS */
 
 a:link, a:visited {
-    color: var(--body-fg);
-    text-decoration: none;
+    color: var(--link-fg);
+    text-decoration: underline;
```

Comment:
@knyghty @thibaudcolas I have one concern about this change, I haven't tested this change yet and can't see the screenshot on my phone, but it got me thinking about titles that can also be links, are they also considered underlined or just the content?

Reply:
As far as I'm aware, all links in the admin will be underlined, so this is a big UI change.

Reply:
Argh, sorry I should have rephrase, I wanted to say : "do we want them also underlined or just the content?"
I don't know if it's already the case according the style all links in the admin will be underlined as you said, but I think we should agree on what we want before to merge this PR